projects/fmcdaq2/src/app/fmcdaq2.c

```diff
@@ -340,7 +340,7 @@ static int fmcdaq2_jesd_init(struct fmcdaq2_init_param *dev_init)
 		.name = "ad9144_xcvr",
 		.base = XPAR_AXI_AD9144_XCVR_BASEADDR,
 		.sys_clk_sel = ADXCVR_SYS_CLK_QPLL0,
-		.out_clk_sel = 4,
+		.out_clk_sel = ADXCVR_REFCLK_DIV2,
```

Comment:
same with this commit. it can be squashed with the one adding the macro definitions